belt-block/src/cipher_impl.rs

```diff
@@ -81,6 +83,61 @@ impl BeltBlock {
             set_u32(block_out, &x, i);
         }
     }
+
+    /// Wide block encryption as described in section 6.2.3.
+    #[inline]
+    pub fn wblock_enc(&self, data: &mut [u8]) {
+        let len = data.len();
+        let n = (len + 15) / 16;
+        // For i = 1,2,...,2n execute:
+        for i in 1..=2 * n {
+            let mut block: Block<BeltBlock> = Default::default();
+            block.copy_from_slice(&data[..16]);
+
+            // 𝑠←𝑟1 ⊕𝑟2 ⊕...⊕𝑟𝑛−1
+            for i in (16..len - 16).step_by(16) {
+                xor_set(&mut block, &data[i..i + 16]);
+            }
```

Comment:
Note that in Rust we usually prefer a more functional approach, which can look something like this:
```rust
type Block = cipher::Block<BeltBlock>;

let s = data.chunks_exact(16).fold(Block::default(), xor);

fn xor(a: Block, b: &[u8]) -> Block { ... }
```

Ideally, `b` would be `Block` as well and instead of `chunks_exact` we would use [`array_chunks`](https://doc.rust-lang.org/std/primitive.slice.html#method.array_chunks), but, unfortunately, const generics are not powerful enough today.